### ISSUE TITLE
Observation table tweaks

### ISSUE THREAD

**Issue by mwinokan** [m2ms/fragalysis-frontend#1508]
@phraenquex I have re-uploaded Flavi_Ns5_RdRp and the CentroidRes is populated in the table. I do now have some questions about the spec. Currently:

![Screenshot 2024-09-09 at 09 14 16](https://github.com/user-attachments/assets/b779d454-99a1-4679-a253-2bb35cabfa90)

|    Category     | Box Tooltip |        Table        |
|-----------------|-------------|---------------------|
| CanonicalSite   | Upload name | Alias + Upload name |
| ConformerSite   | Upload name | Upload name         |
| CrystalformSite | Upload name | Upload name         |
| Crystalform     | Upload name | Upload name         |
| Quatassembly    | Upload name | Upload name         |
| CentroidRes     | N/A         | LongCode            |
| LongCode        | N/A         | LongCode            |

1. Hovering over the colourful site boxes shows the original upload name of the generated tags. E.g. `Zika_NS5A-x0411+A+1104+1`, shouldn't we show the alias assigned by the curator instead?
2. The table only shows the original upload name and alias pair for the CanonicalSites tags, the other categories either show the only show the upload name. Should we have a toggle to decide between showing the alias or upload name for the observation table view?

**Comment by phraenquex** [comment]
@mwinokan do you know why the shortcodes weren't assimilated into the Site names?  Or is that ticket still ongoing?

**Comment by mwinokan** [comment]
They are assigned as the default value for the alias. But that's currently only shown in the 'Edit Tags' dialog

**Comment by phraenquex** [comment]
Yes, but all site aliases are meant to be generated using the short code - though I'm not sure what ticket that was captured in.  Maybe #1482. 

@kaliif can you comment please - including whether we ever actually put that into a ticket.

**Comment by kaliif** [comment]
All tags have 3 versions now: long name, short name, and alias. All are being stored in the database and served by the API endpoint `api/siteobservation_tag/`. Cannot comment on what happens in the frontend.

**Comment by phraenquex** [comment]
- [ ] Change column order:  `QuatAssembly`, `ConformerSite`, `CrystalForm`, `CrystalFormSite`, `LongCode`, `TagName`, `CentroidRes`, `CanonSite`
- [ ] Panel height behaviour is shonky:  should be full-height even if that stretches below the bottom edge (currently it gets a scroll bar if the hit is scrolled too low - and that only goes away if extending or shrinking)
- [ ] Format for tags & centroid etc should be only slashes (instead of `_` and `+`), e.g. A0300/A/141/A/v1  **(Loader feature)**
- [ ] Long code:  is alternate code avaible to include?  Check yaml  **(Loader feature)**
- [ ] Tooltips are wrong:  show long code  (currently just `Centroid` or something generic)
- [ ] Shortcode tooltip:  show short code text
- [ ] CrystalFormSite mis-assigned (loader) or mis-displayed (f/e) (see e.g. CHIKV c0399*)
- [ ] ConformerSite may be wrong too - double-check the assignment rules (loader & f/e)
- [ ] How does one move observations into a different pose?  (Only have button for "new pose")

@mwinokan please add your questions to this list (edit in place).

**Comment by mwinokan** [comment]
### Re: CrystalFormSite mis-assigned (loader) or mis-displayed (f/e) (see e.g. CHIKV c0399*)

CrystalFormSites and (possibly) ConformerSites should not be the same across all seven of these observations. Need to check against the YAML's from XCA and determine whether it is a loader or f/e issue.

![Screenshot 2024-09-10 at 12 10 25](https://github.com/user-attachments/assets/dd04b2b9-424b-4348-ac99-8fbd431ff0ad)

**Comment by matej-vavrek** [comment]
@phraenquex, @mwinokan, F/E changes were deployed on [my stack](https://fragalysis-matej-default.xchem-dev.diamond.ac.uk/). It cause some error (investigating) with latest B/E when loading target so you need to select some tag.

Currently, when we show aliases for columns then `CanonSite` and `TagName` are the same because they show `CanonSite` alias / editable tag name.

**Comment by mwinokan** [comment]
@kaliif has updated the tag strings to be `/` delimited, and I can confirm the [behaviour is correct on staging](https://fragalysis.xchem.diamond.ac.uk/viewer/react/projects/67/38). 

@matej-vavrek I will look at your stack now

**Comment by mwinokan** [comment]
@matej-vavrek please remove the TagName column as discussed on Slack and then push to staging so we can compare with the latest b/e

**Comment by matej-vavrek** [comment]
Based on discussion with @mwinokan, changed the order of the columns in the observation header to be:

- Quatassembly
- ConformerSite
- Crystalform
- CrystalformSite
- CanonSite
- CentroidRes
- LongCode

Changed the order of the colourful boxes in the observations panel to be:

- CanonSite (unchanged)
- ConformerSite (unchanged)
- Quatassembly
- Crystalform
- CrystalformSite
